examples/components/styles.js

```diff
@@ -1,6 +1,7 @@
 import '../../src/more-or-less/moreOrLess.scss';
 import '../../src/flex/flexView.scss';
 import '../../src/loading-spinner/style.scss';
+import '../../src/dropdown/dropdown.scss';
 import '../../src/confirmation-input/confirmationInput.scss';
 import '../../src/badge/badge.scss';
 import '../../src/toggle/toggle.scss';
```

Comment:
you should remove import of the old style: `import '../../node_modules/react-select/dist/default.css';`
(it's two lines below but I can't comment on it)